### DISCUSSION TITLE
How to Change Phase

### DISCUSSION THREAD

**Discussion by happtain** [InteractiveComputerGraphics/SPlisHSPlasH#318] [Q&A]
Hello，
I noticed that SPlisHSPlasH could simulate deformable solids. Is there any method or interface could to implement phase change like fluid to solid?

Best Regards.

**Comment by janbender** [comment]
At the moment this is not possible in SPlisHSPlasH. If you want to implement something like this, you would have to take care to initialize the solid particles accordingly.
